### ISSUE TITLE
AWS::ServiceCatalog::CloudFormationProvisionedProduct.Outputs

### ISSUE THREAD

**Issue by purvish14** [aws-cloudformation/cfn-python-lint#1931]
cfn-lint version: 0.47.0

Given the following example template:
```yaml
AWSTemplateFormatVersion: '2010-09-09'
Description: Serverless Stack
Resources:
    SimpleLambda:
      Type: AWS::ServiceCatalog::CloudFormationProvisionedProduct
      Properties:
        ProductName: Basic Lambda
        ProvisioningArtifactName: '1.0'
      
    SimpleApiGateway:
      Type: AWS::ServiceCatalog::CloudFormationProvisionedProduct
      Properties:
        ProductName: API Gateway
        ProvisioningArtifactName: '1.0'
        ProvisioningParameters:
            -
              Key: LambdaArn
              Value: !GetAtt [SimpleLambda, Outputs.SCLambdaArn]
```

The following linting errors are thrown:
```
E1010 Invalid GetAtt SimpleLambda.Outputs.SCLambdaArn for resource SimpleApiGateway
product.template.yaml:18:15
```

This issue was fixed in #1659 PR, but I'm still getting this error.

Documentation: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-servicecatalog-cloudformationprovisionedproduct.html#aws-resource-servicecatalog-cloudformationprovisionedproduct--examples

**Comment by kddejong** [comment]
Its because the Outputs were dropped from the resource spec. I'm still waiting on a good answer if that drop was intentional or not.

**Comment by kddejong** [comment]
Also worth noting that `Outputs` was dropped from the [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-servicecatalog-cloudformationprovisionedproduct.html#aws-resource-servicecatalog-cloudformationprovisionedproduct-return-values) above that example.

**Comment by purvish14** [comment]
I have tested a cloudformation template which uses GetAtt function to refer outputs. It still works.

**Comment by kddejong** [comment]
Thanks.  I put in a patch to put it back in until the resource spec is fixed.

**Comment by kddejong** [comment]
released with https://pypi.org/project/cfn-lint/0.47.1/
